src/middlewared/middlewared/plugins/jail.py

```diff
@@ -582,12 +582,12 @@ def start(self, jail):
 
         return True
 
-    @accepts(Str("jail"))
-    def stop(self, jail):
+    @accepts(Str("jail"), Bool('force'))
```

Comment:
Can we have a default value here? (also in master)

Reply:
Ugh, good catch. Let me make a special PR for master